constants/lib/constants.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Constants do
```

Comment:
Don't change this, but you could have also put these in config/..., which would let you change them without a recompile...

Reply:
Gotcha - I'll open an issue related to this and the pointless clock module on github so I remember to fix them after the class - low priority